python/pyspark/serializers.py

```diff
@@ -220,6 +220,29 @@ def __repr__(self):
         return "BatchedSerializer(%s, %d)" % (str(self.serializer), self.batchSize)
 
 
+class FlattedValuesSerializer(BatchedSerializer):
```

Comment:
Can we rename this to `FlattenedValuesSerializer` (with the missing `en`)?  Should be a pretty simple find-and-replace, but we should also update the other occurrences of `flatted` to be `flattened`.